ibm_catalog.json

```diff
@@ -154,6 +154,194 @@
+                        "authority": "scc-v3",
+                        "profiles": [
+                            {
+                                "profile_name": "IBM Cloud Framework for Financial Services",
+                                "profile_version": "1.5.0"
+                            }
+                        ]
+                    },
+                    "architecture": {
+                        "features": [
+                            {
+                                "title": "Secure cross region Bucket",
+                                "description": "Create a Secure cross region Bucket."
+                            },
+                            {
+                                "title": "Configure bucket retention",
+                                "description": "Configure retention policies at the IBM Cloud Object Storage bucket level."
+                            },
+                            {
+                                "title": "Enable object versioning",
+                                "description": "Supports enabling object versioning to keep multiple versions of an object in a bucket."
+                            },
+                            {
+                                "title": "Record Activity Tracker events",
+                                "description": "Integrate Activity Tracker to track events."
+                            },
+                            {
+                                "title": "Monitor data",
+                                "description": "Integrate with the IBM Cloud-Monitoring service to monitor your Object Storage data."
+                            },
+                            {
+                                "title": "Encrypt data with key management services",
+                                "description": "Supports key management encryption (BYOK and KYOK), including service-to-service authorization policies with KMS."
+                            }
+                        ],
+                        "diagrams": [
+                            {
+                                "diagram": {
+                                    "caption": "IBM Cloud Object Storage Solution",
+                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-cos/main/reference-architectures/secure-cross-regional-bucket.svg",
+                                    "type": "image/svg+xml"
+                                },
+                                "description": "This architecture supports creating and configuring an IBM Cloud Object Storage instance and a cross region bucket."
+                            }
+                        ]
+                    }
+                },
+                {
+                    "label": "Secure Regional Bucket",
+                    "name": "secure-regional-bucket",
+                    "working_directory": "solutions/secure-regional-bucket",
```

Comment:
This is an extension type DA, so you need to add the `install_type` and `dependencies` section to the JSON. 

```
         "install_type": "extension",
          "dependencies": [
            {
              "flavors": [
                "instance"
              ],
              "id": "68921490-2778-4930-ac6d-bae7be6cd958",
              "name": "deploy-arch-ibm-cos",
              "version": ">= v1.0.0"
            }
          ]
```